Modules/Brand/Http/Services/BrandService.php

```diff
@@ -206,15 +228,63 @@ public function getShop(string $brandKey): array
         $brand->tag_shop_page = $brand->tag_shop_page != '' ? explode(',', $brand->tag_shop_page) : array();
 
         //country
-        $country = Country::where('id', $brand->country)->first();
+        $country = Country::where('id', $brand->country)->first(['name']);
         $brand->country = $country->name;
         //headquarter
-        $headquarteredCountry = Country::where('id', $brand->headquatered)->first();
+        $headquarteredCountry = Country::where('id', $brand->headquatered)->first(['name']);
         $brand->headquatered = $headquarteredCountry->name;
         //shipped from
-        $productShippedCountry = Country::where('id', $brand->product_shipped)->first();
+        $productShippedCountry = Country::where('id', $brand->product_shipped)->first(['name']);
```

Comment:
Make an array of the id and then use whereIn to get all theree courntries like
$requiredCountries = Countries::whereIn('id',$arrayOfIds)->get();
then get the single country form $requiredCountries.